_maps/shuttles/shiptest/scoutcourier.dmm

```diff
@@ -0,0 +1,2155 @@
+/obj/item/reagent_containers/food/snacks/canned/beans,
+/obj/item/reagent_containers/food/snacks/canned/beans,
+/obj/item/reagent_containers/food/snacks/canned/beans,
+/obj/effect/spawner/lootdrop/maintenance/two,
+/obj/effect/turf_decal/box,
+/turf/open/floor/plasteel/tech/grid,
+/area/ship/cargo)
+"bn" = (
+/obj/structure/sign/poster/retro/we_watch,
+/turf/closed/wall/mineral/plastitanium,
+/area/ship/bridge)
+"bI" = (
+/obj/effect/decal/cleanable/oil,
+/obj/effect/decal/cleanable/glass,
+/obj/machinery/autolathe,
+/turf/open/floor/plating,
+/area/ship/engineering)
+"bO" = (
+/turf/closed/wall/mineral/plastitanium,
+/area/ship/bridge)
+"co" = (
+/obj/structure/closet/wall{
+	pixel_y = -28;
+	dir = 1
+	},
+/obj/item/storage/firstaid,
+/obj/item/storage/firstaid,
+/obj/effect/turf_decal/corner/opaque/syndiered/bordercorner{
+	dir = 4
+	},
+/turf/open/floor/plasteel/dark,
+/area/ship/crew)
+"cu" = (
+/obj/machinery/door/airlock/external,
+/obj/effect/turf_decal/trimline/opaque/syndiered/filled/warning,
+/obj/effect/turf_decal/trimline/opaque/syndiered/filled/warning{
+	dir = 1
+	},
+/obj/docking_port/mobile{
+	callTime = 250;
+	dir = 2;
+	launch_status = 0;
+	name = "Scout Courier";
+	port_direction = 8;
+	preferred_direction = 4
+	},
+/turf/open/floor/plasteel/white,
+/area/ship/engineering)
+"cw" = (
+/obj/machinery/atmospherics/components/unary/vent_scrubber/on/layer4{
+	dir = 4
+	},
+/turf/open/floor/plasteel/tech/techmaint,
+/area/ship/cargo)
+"cL" = (
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
+	dir = 4
+	},
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{
+	dir = 8
+	},
+/obj/structure/cable{
+	icon_state = "4-8"
+	},
+/turf/open/floor/plasteel/white,
+/area/ship/engineering)
+"cY" = (
+/obj/machinery/porta_turret/centcom_shuttle/weak,
+/turf/closed/wall/mineral/plastitanium,
+/area/ship/bridge)
+"cZ" = (
+/turf/closed/wall/mineral/plastitanium/nodiagonal,
+/area/ship/cargo)
+"df" = (
+/obj/machinery/atmospherics/pipe/manifold/supply/hidden/layer2,
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{
+	dir = 8
+	},
+/obj/structure/cable{
+	icon_state = "4-8"
+	},
+/obj/structure/cable{
+	icon_state = "2-8"
+	},
+/obj/effect/turf_decal/trimline/opaque/bar/filled/warning,
+/turf/open/floor/plasteel/white,
+/area/ship/hallway/central)
+"dj" = (
+/obj/item/toy/cards/deck,
+/obj/machinery/light/small/broken{
+	dir = 1
+	},
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{
+	dir = 10
+	},
+/turf/open/floor/plating,
+/area/ship/engineering)
+"dq" = (
+/obj/structure/window/reinforced/spawner/north,
+/obj/effect/turf_decal/box,
+/turf/open/floor/plasteel/tech/grid,
+/area/ship/cargo)
+"dx" = (
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{
+	dir = 5
+	},
+/obj/effect/turf_decal/corner/opaque/syndiered/bordercorner{
+	dir = 4
+	},
+/turf/open/floor/plasteel/dark,
+/area/ship/crew)
+"dA" = (
+/obj/machinery/door/airlock/command{
```

Comment:
Decal work in some areas of this ship could use some work. I'd recommend adding decals in front of certain doors, particularly the bridge and engineering. Additionally, replace the floors underneath the airlocks with tech floors instead of white tiles. The slight change in texture really sells the transition from room to room. Consider changing the floors underneath the bridge consoles to tech floors or telecoms floors as well to add some visual interest.

Reply:
put some more detail into engi and the command hallway.
![StrongDMM_Sxd75jtpRl](https://user-images.githubusercontent.com/94164348/185770656-fba7c7a2-b05d-4c57-beb4-d0213b9167a4.png)
 
![StrongDMM_BxscPmSlUt](https://user-images.githubusercontent.com/94164348/185770659-073be428-ca34-4a48-96ad-a95ba6dd5ff4.png)